### PR TITLE
docs: added link to CloudFormation template for VPC

### PR DESCRIPTION
**Pull request by onhate**
Added link to CloudFormation template for VPC creation with Internet access via NAT.

### CODE REVIEW

**Comment by rpaik** [comment]
@hassankhan can you help with the review?

File: docs/content/Deployment/Deployment-Guide.md [hunk 1]
```diff
@@ -173,10 +173,13 @@ provider:
 When you assign a VPC to a Lambda function, internet access will be disabled by
 default. Lambda functions require internet access to send SNS messages for query
 processing. Please follow [this guide][link-vpc-internet] to set up internet
-access for your Lambda functions.
+access for your Lambda functions or use [this cloudformation template][link-vpc-cf-template]
```
Comment (hassankhan) [inline docs/content/Deployment/Deployment-Guide.md:176]:
```suggestion
access for your Lambda functions or use [this template][link-vpc-cf-template]
```

File: docs/content/Deployment/Deployment-Guide.md [hunk 2]
```diff
@@ -173,10 +173,13 @@ provider:
...
+to create the VPC, Subnets and NAT configuration using CloudFormation.
```
Comment (hassankhan) [inline docs/content/Deployment/Deployment-Guide.md:177]:
```suggestion
to create the VPC, subnets and NAT configuration using CloudFormation.
```

**Review by hassankhan — CHANGES_REQUESTED** [review]
Great work @onhate :tada:

If you could fix the minor nitpicks then I'm happy to merge 👍

**Review by hassankhan — APPROVED** [review]

**Comment by hassankhan** [comment]
Thank you for your contribution @onhate 🙌

**Pull request merged**